### PR TITLE
dmirano/4509-outcomes-and-metrics-help-text

### LINKED ISSUE #4509: [Feature] Help Text Change for Outcomes and Metrics
### Description and related issues
Based on design ticket #4381 

### Mockups or link to Figma
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3328%3A18573&t=VXupikAaT7lnGE7V-1

Outcomes and Metrics

### Acceptance criteria

- [ ] change the example box to
> **CMS Required Outcome Example:**
> Outcome: A provider can initiate, save, and apply to be a Medicaid provider.
> Metrics:
>1. Number of requests to help desk for problems with initiating, saving, and applying.
>2. Average time to enroll from point of submit.
>  a. Total time to enroll all providers
>  b. Total # of enrolled providers
>
> **State Specific Outcome Example**
> Outcome: Transform MHD from a transactional, request as needed data approach to a business intelligence driven model recognizing and treating data as an asset.
> Metrics:
>1. Conducting an initial data maturity analysis and prioritizing recommendations and review every two (2) years.
>2. Establishing a Data Governance Council who is engaged and meets at a minimum of quarterly and has at least 60% participation as measured by attendance.
>
>To view other outcome and metrics examples, visit the [CMS Certification Repository.](https://cmsgov.github.io/CMCS-DSG-DSS-Certification/)

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #4509 

### Description
Changes the help text for the outcomes and metrics page for mmis only.

### Significant changes or possible side effects
Only affects outcomes.js and the corresponding yaml file

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Verify HITECH has old help text
2. Verify MMIS has the new help text

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [ ] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

**Review by tbolt — APPROVED** [review]

File: web/src/pages/apd/activities/oms/Outcomes.test.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ describe('activity Outcomes and Metrics component', () => {
     const state = {
       apd: {
         data: {
+          apdType: 'HITECH',
```
Comment (thetif) [inline web/src/pages/apd/activities/oms/Outcomes.test.js:51]: you should use the APD_TYPE.HITECH const here

File: web/src/pages/apd/activities/oms/Outcomes.test.js [hunk 2]
```diff
@@ -60,6 +61,7 @@ describe('activity Outcomes and Metrics component', () => {
     const ownProps = { activityIndex: 2 };
 
     expect(mapStateToProps(state, ownProps)).toEqual({
+      apdType: 'HITECH',
```
Comment (thetif) [inline web/src/pages/apd/activities/oms/Outcomes.test.js:64]: you should use the APD_TYPE.HITECH const here

**Review by thetif — APPROVED** [review]

**Review by jeromeleecms — CHANGES_REQUESTED** [review]
Can you make the link to the CMS certification repository open in a new tab? Everything else looks good!

**Review by jeromeleecms — APPROVED** [review]
Looks good!

**Review by SGilliamA1M — APPROVED** [review]
Looks good!

**Pull request merged**